src/test/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ Set(${dir_INVOLVED_TEST_EXE_SOURCES}
         # the next 2 are interactive, so we don't add a test for it, but only compile them
 	test_display
 	test_interpolate
+        test_Transformations
```

Comment:
tricky where this file should be. I guess in future really `test/motion` or similar. For now, we'll keep these classes in `local`, therefore should be in `local/test/`.

Name isn't ideal. what about `test_BSpline_transformations`?